packages/logo/logo.scss

```diff
@@ -6,6 +6,9 @@ $jkl-logo-magic-origin--f: 30px 132px; /* Is connected to the viewbox of the log
 $jkl-logo-magic-origin--line: 34.3px; /* Must be the same as the line x1 origin, change at your own risk */
 
 .jkl-logo {
+    width: 100%;
+    height: 100%;
```

Comment:
This can only be a temporary fix: We don't really want to set height and width in the stylesheet, as it makes it more difficult to set when using the component. It would be more optimal to fix this in the example IMO. If that necessitates changes to the component, we can merge this as a stopgap measure while we figure out the "real" change 😊

Reply:
ready to merge